src/features/stickyMessage/messageCooldown.spec.ts

```diff
@@ -0,0 +1,112 @@
+import { Message } from 'discord.js'
+
+import { StickyMessage } from '@prisma/client'
+import { afterEach, describe, expect, it, vi } from 'vitest'
+
+import * as cache from '../../utils/cache.js'
+
+import * as stickyMessage from './index.js'
+import { resetCooldown } from './messageCooldown.js'
+import * as messageCounter from './messageCounter.js'
+
+vi.mock('../../config.js', async () => {
+  const Environment = {
+    MESSAGE_COOLDOWN_SEC: 15,
+    MESSAGE_MAX: 5,
+  }
+
+  return { Environment }
+})
```

Comment:
i see that you try to mock the config file in each test file. this is very repetitive and prone to error. there is a mock file at `@/__mocks__/config.ts`, maybe you can add it there instead.